backend/Application/AppGenerator.php

```diff
@@ -51,7 +52,7 @@ public function generateStaticData(): void
         $this->saveDataToJson($this->facultyFactory, $faculties);
         foreach ($faculties as $faculty) {
             foreach ($this->subFactories as $subFactory) {
-                echo "Processing " . $faculty->getDirectory() . "." . PHP_EOL;
+                OutputWriter::newLineToConsole("Processing " . $faculty->getDirectory());
```

Comment:
```suggestion
                OutputWriter::newLineToConsole("Processing  {$faculty->getDirectory()}");
```

https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.single

Reply:
Oh! Interesting, I wasn't aware of that. I'll apply the changes to the whole project.